packages/components/pagination/src/components/pager.vue

```diff
@@ -141,6 +149,13 @@ function onMouseenter(direction: 'left' | 'right') {
     quickNextHover.value = true
   }
 }
+function onFocus(direction: 'left' | 'right') {
```

Comment:
**Suggestion:** when it comes to a flag with only 2 values, we should use `boolean` instead of `string` here, for instance: 

```ts
function onFocus(forward = false) {
}
```

Reply:
@jw-foss Thank you review!


As you said, there are only 2 flags, so I thought it would be appropriate to treat it as a boolean. However, there is one more logic like this. For example, can I change the corresponding part of `onMouseenter(direction: 'left' | 'right')` in the same way?

🙇🙇

Reply:
And you recommended `forward`. But when I thought about it, I thought ‘backward’ was appropriate.

The reason is that 'forward' usually means moving forward, so it's likely that pagination means moving to the right. However, if this value is true, the actual motion is shifted to the left.

What do you think about this?